.github/workflows/run-e2e-tests.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
     permissions:
       contents: read
       id-token: write
-    uses: broadinstitute/dsp-reusable-workflows/.github/workflows/run_tsps_e2e_tests.yaml@main
+    uses: broadinstitute/dsp-reusable-workflows/.github/workflows/run_teaspoons_e2e_tests.yaml@main
```

Comment:
we'll need to open a PR in dsp-reusable-workflows to update the names of all the relevant files - they all say "tsps" currently, e.g. https://github.com/broadinstitute/dsp-reusable-workflows/blob/main/.github/workflows/run_tsps_e2e_tests.yaml